ai/ollama/v0/config/tasks.json

```diff
@@ -0,0 +1,337 @@
+        }
+      },
+      "required": [
+        "role",
+        "content"
+      ],
+      "title": "Chat Message",
+      "type": "object"
+    },
+    "usage": {
+      "description": "Usage tokens in Anthropic",
+      "instillUIOrder": 1,
+      "properties": {
+        "input-tokens": {
+          "description": "The input tokens used by Anthropic",
+          "instillFormat": "number",
+          "instillUIOrder": 2,
+          "title": "Input Tokens",
+          "type": "number"
+        },
+        "output-tokens": {
+          "description": "The output tokens used by Anthropic",
+          "instillFormat": "number",
+          "instillUIOrder": 3,
+          "title": "Output Tokens",
+          "type": "number"
+        }
+      },
+      "required": [
+        "input-tokens",
+        "output-tokens"
+      ],
+      "title": "Usage",
+      "type": "object"
+    }
+  },
+  "TASK_TEXT_GENERATION_CHAT": {
+    "instillShortDescription": "Provide text outputs in response to text/image inputs.",
+    "description": "Open-source large language models (OSS LLMs) are artificial intelligence models with publicly accessible code and architecture, allowing for free use, modification, and distribution. These models can provide performance comparable to proprietary alternatives. Ollama is a tool that enables you to run and interact with OSS LLMs using limited computational resources. You can install Ollama from: https://ollama.com.",
+    "input": {
+      "description": "Input",
+      "instillEditOnNodeFields": [
+        "prompt",
+        "model-name"
+      ],
+      "instillUIOrder": 0,
+      "properties": {
+        "chat-history": {
+          "description": "Incorporate external chat history, specifically previous messages within the conversation. Please note that System Message will be ignored and will not have any effect when this field is populated. Each message should adhere to the format: : {\"role\": \"The message role, i.e. 'system', 'user' or 'assistant'\", \"content\": \"message content\"}.",
+          "instillAcceptFormats": [
+            "structured/chat-messages"
+          ],
+          "instillShortDescription": "Incorporate external chat history, specifically previous messages within the conversation.",
+          "instillUIOrder": 4,
+          "instillUpstreamTypes": [
+            "reference"
+          ],
+          "items": {
+            "$ref": "#/$defs/chat-message"
+          },
+          "title": "Chat history",
+          "type": "array"
+        },
+        "max-new-tokens": {
+          "default": 50,
+          "description": "The maximum number of tokens for model to generate",
+          "instillAcceptFormats": [
+            "integer"
+          ],
+          "instillUIOrder": 6,
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "title": "Max new tokens",
+          "type": "integer"
+        },
+        "model-name": {
```

Comment:
nit
Today, @GeorgeWilliamStrong mentioned about we should make this field consistent.
Let's stick it to model from now on!

If you have time, please also help us make all ai component to model from model-name.

Reply:
Fixed